ci-operator/config/open-cluster-management/prometheus/open-cluster-management-prometheus-release-2.4.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.15-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  from: base
```

Comment:
Since you have the full path to the builder image in your Dockerfile.prow, you don't need to specify the from field.

However, in your Dockerfile.prow, you may want to consider using the nodejs builder image instead of the Go builder.

Change this line:
https://github.com/open-cluster-management/prometheus/blob/release-2.4/Dockerfile.prow#L3

to 
```
FROM registry.ci.openshift.org/open-cluster-management/builder:nodejs14-linux AS builder
```

That image already has nodejs and yarn installed which will save time in your build.

Reply:
thanks, but we just want to build a Go binary. so I think we need this builder img: `registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux`